tvdbsimple/series.py

```diff
@@ -134,6 +131,10 @@ def __init__(self, id, language='', **kwargs):
         self._set_language(language)
         self.update_filters(**kwargs)
 
+        self._PAGES = -1
+        self._PAGES_LIST = {}
+        self._FILTERS = {}
```

Comment:
You're overwriting possible filters set by update_filters... I'll merge https://github.com/phate89/tvdbsimple/pull/3/files that fixes the same problem and it's an older pr.. can you drop the change?

Reply:
good catch. done.